wagon-provider-api/src/main/java/org/apache/maven/wagon/authentication/AuthenticationInfo.java

```diff
@@ -51,7 +52,300 @@
      * The absolute path to private key file
      */
     private String privateKey;
+    
+     /**
+     *
```

Comment:
Nit-picking: I don't like the superfluous `*` lines. Also, I think the Javadoc specs say they shouldn't be there.

Reply:
Same here

Reply:
np, someone should tell the netbeans folks then. auto formatter does that. If it bothers you all that much, add it to the check style xml file

Reply:
actually i think it was from the maven core mdo > java class generator that did that and i just copied and pasted... no worries